src/features/management/likeroles.js

```diff
@@ -0,0 +1,187 @@
+import channel from '../../internal/channel.js'
+import colors from '../../internal/colors.js'
+import event from '../../internal/event.js'
+import groups from '../../internal/groups.js'
+import P from '../../internal/permissions.js'
+import { register } from '../../internal/register.js'
+import { path } from '../../utils/object.js'
+import reference from '../../utils/reference.js'
+import { error, message } from '../../utils/response.js'
+import { verifyRolePosition } from '../../utils/role.js'
+import features from '../index.js'
+import { man } from '../settings/man.js'
+
+register('likeroles', event.onReaction, channel.text)
+
+export default {
+    name: 'likeroles',
+    group: groups.management,
+    description: 'likeroles.description',
+    usage: `likeroles \`create|message_id\`
+    [emoji] [@role] [description]
+    [emoji] [@role] [description]`,
+    examples: 'likeroles.examples',
+    arguments: 2,
+    permissions: [P.MANAGE_ROLES],
+
+    execute: async (context, action, ...args) => {
+
+        if (!action) {
+            return man(context, 'likeroles')
+        }
+
+        if (action !== 'create' && isNaN(action)) {
+            return man(context, 'likeroles')
+        }
+
+        const msg = action === 'create' ? null : await context.channel.messages.fetch(action);
+
+        if (args.length === 0) {
+            return man(context, 'likeroles')
+        }
+
+        const lines = [] 
+        args[0].trimStart('\n').split('\n').forEach((line) => {
+            let elements = line.split(' ').filter(x => x !== '')
+
+            if (elements.length < 2) {
+                return;
+            }
+
+            lines.push({
+                emoji: elements[0],
+                role: elements[1],
+                description: elements.slice(2).join(' ') || ''
+            })
+        })
+
+        if (lines.length === 0) {
+            return man(context, 'likeroles')
+        }
+
+        var fields = [ getInitialField() ]
+        var emojis = []
+
+        await Promise.all(lines.map(async obj => {
+            let emoji = await getEmoji(context, obj.emoji)
+            let role = await getRole(context, obj.role)
+            let description = obj.description
```

Comment:
better use `const` instead of let if you don't have an intention to change these properties.